Client/game_sa/CFxSA.h

```diff
@@ -49,13 +50,17 @@ class CFxSA : public CFx
     void AddPunchImpact(CVector& vecPosition, CVector& vecDirection, int);
     void AddDebris(CVector& vecPosition, RwColor& rwColor, float fScale, int iCount);
     void AddGlass(CVector& vecPosition, RwColor& rwColor, float fScale, int iCount);
+    bool AddShadow(eShadowType shadowType, CVector& vecPosition, CVector2D& vecOffset1, CVector2D& vecOffset2, SColor color, float fZDistance,
+                   bool bDrawOnWater, bool bDrawOnBuildings);
     void TriggerWaterHydrant(CVector& vecPosition);
     void TriggerGunshot(CEntity* pEntity, CVector& vecPosition, CVector& vecDirection, bool bIncludeSparks);
     void TriggerTankFire(CVector& vecPosition, CVector& vecDirection);
     void TriggerWaterSplash(CVector& vecPosition);
     void TriggerBulletSplash(CVector& vecPosition);
     void TriggerFootSplash(CVector& vecPosition);
 
+    bool IsShadowsLimitReached();
 private:
     CFxSAInterface* m_pInterface;
+    std::unordered_map<eShadowType, RwTexture*> m_textureMap;
```

Comment:
It seems that **m_textureMap** is not actually used. At least I see the creation of a texture within a map meaningless because it's not used further down the code.